### DISCUSSION TITLE
DAT Linux Admin does not works & Apply Full Upgrade neither

### DISCUSSION THREAD

**Discussion by joseADuranC** [dat-linux/community#52] [Issues]
Greetings dear creators:

Less than 1 day ago I ran into the following problem when using Apps Updater:
___________________________
📥️ DAT Linux App Updater.
-----------------------------------------------
/usr/local/bin/datlinux/dat-update: line 16: cari: command not found
/usr/local/bin/datlinux/dat-update: line 17: cari: command not found
Nothing to update.. \()/
___________________________
Additionally, DAT-Sysinfo does not show the list of installed programs either, and Apps Unistaller also throws a similar error with the 'cari' command.

Also, when I run Tilix, I am forced to type the 'bash' command or the 'zsh' command for said Terminal to work correctly.

All of the above happened to me when I was doing two tasks:
1/ follow these instructions to learn Rust language from my PC
https://exercism.org/cli-walkthrough
2/ update rust and enable clippy
https://code.visualstudio.com/docs/languages/rust#_linting
I accomplished both tasks satisfactorily, but my PC fails with everything I have told you.

In light of all of the above, how do I repair my described problem? All these problems were what I found in 1 day.
First of all, Thanks.
judasnetwork

**Comment by dat-linux** [comment]
Hi, it looks like something has possibly gone wrong with your home directory.

Please run `/usr/bin/bash -c "which cari"` and report results. 

Also, be aware if you changed your default SHELL from bash, as .bashrc needs to be sourced. And $HOME/.cari/bin/cari needs to be on the PATH.

Re tilix, not sure why you're having a problem, I just installed it and works fine. Clearly this is related to whatever caused the cari issue in the first place.

**Reply by joseADuranC** [reply]
Hello! 
Nothing happens when I run `/usr/bin/bash -c "which cari"`:

![image](https://github.com/dat-linux/community/assets/34608403/9e719b4e-d641-4a71-b674-170f997ba71e)

Apologies, but I don't understand when you talk to me about .bashrc. How do I add `$HOME/.cari/bin/cari` to the PATH?

Yes, Tilix certainly worked well before, the `cari` issue caused the damage on this one.

Atento a tu amable respuesta. 
Judas

**Reply by ghost** [reply]
the same problem I have, how to solve it?

**Reply by joseADuranC** [reply]
Hello Bastaan!
Can you tell us about what event or action the same problem happened to you? I suspect that in my case it was when updating the Rust programming language.
I still have the same problem. Would it be convenient to uninstall and install the new DAT Linux 1.0.2.1 version?

**Comment by joseADuranC** [comment]
Decidí reinstalar la última versión de DAT Linux y éste lo hizo con errores, luego decidí migrar a otro sistema operativo. 
Ya No requiero la solución de éste problema. Felicidades.
